apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -137,7 +146,7 @@ export function ListBoxInline({ app, fieldName, stateName = '$', options = {} })
         <Grid item>
           {showTitle && (
             <Typography variant="h6" noWrap>
-              {layout.title || fieldName}
+              {!isMaster ? layout.title || fieldName : layout.qListObject.qDimensionInfo.qFallbackTitle}
```

Comment:
have you considered if setting title should override the title from the master dimension?
(if yes you could use `layout.title || layout.qListObject.qDimensionInfo.qFallbackTitle`